ethsigner/signer/multikey/src/main/java/tech/pegasys/ethsigner/signer/multikey/SigningMetadataTomlConfigLoader.java

```diff
@@ -152,28 +151,18 @@
   }
 
   private Optional<SigningMetadataFile> getHashicorpMetadataFromToml(
-      final String filename, final TomlParseResult result) {
+      final Path inputFile, final TomlParseResult result) {
+
+    final String filename = inputFile.getFileName().toString();
 
     final Optional<TomlTableAdapter> signingTable = getSigningTableFrom(filename, result);
     if (signingTable.isEmpty()) {
       return Optional.empty();
     }
 
-    final HashicorpConfig.HashicorpConfigBuilder builder =
-        new HashicorpConfig.HashicorpConfigBuilder();
-    final TomlTableAdapter table = signingTable.get();
+    final HashicorpKeyConfig config = TomlConfigLoader.fromToml(inputFile, "signing");
```

Comment:
This could raise a HashicorpException so think we should the same pattern elsewhere in this class and catch the exception log and return an Optional.empty

Reply:
This can throw, but will be caught in "getMetadataInfo" where it has a blanket exception catch and shows "Could not load TOML File <>" - is that good enough?
Other file loads let their exceptions get caught in the same function as far as I can tell

Reply:
I think that's fine